package.json

```diff
@@ -49,8 +49,9 @@
   "dependencies": {
     "autosize": "3.0.21",
     "downshift": "1.26.1",
-    "react-flatpickr": "3.6.3",
+    "file-loader": "false1.1.6",
```

Comment:
Not sure if this is correct version

Reply:
I don't know why the version looked like this. Even when I tried to fix it by re-running `yarn add file-loader` , the updated versioning was `false1.1.8`. Super weird, but I just deleted the `false` and it worked fine. So we'll just go with that.